src/lib/io/sol-iio.c

```diff
@@ -43,6 +43,7 @@
 #endif
 
 struct sol_iio_device {
+    double *mount_matrix;
```

Comment:
I see that the size is constant. Why not change it to: double mount_matrix[MOUNT_MATRIX_LEN]; ?